src/main/kotlin/net/maizegenetics/phgv2/cli/Phg.kt

```diff
@@ -37,7 +39,9 @@ class Phg : CliktCommand() {
 
 fun main(args: Array<String>) = Phg()
     .subcommands(SetupEnvironment(), Initdb(),  CreateRanges(), AnnotateFastas(), AgcCompress(), AlignAssemblies(),
-        CreateRefVcf(), CreateMafVcf(), LoadVcf(), ExportVcf(), CreateFastaFromHvcf(), StartServer(),
-        HaploidPathFinding(), DiploidPathFinding()
+        CreateRefVcf(), CreateMafVcf(), LoadVcf(), ExportVcf(),
+        BuildKmerIndex(), MapKmers(), HaploidPathFinding(), DiploidPathFinding(), //Imputation
+        CreateFastaFromHvcf(), StartServer() //Utilities
```

Comment:
Given the amount of commands that the CLI has now, I wonder if we should look into ways to group each of these commands in the output.

Reply:
I agree.  I will add this to the trello board to investigate further.